src/services/storage/drivers/relational/index.ts

```diff
@@ -111,32 +111,32 @@ export class RelationalStorage implements StorageDriver {
 
 		public async save(urlBody: UrlRequestData): Promise<StoredUrl> {
 			const url = urlBody.url
-			const urlInfo: Partial<UrlWithInformation> = {
+			const urlInfo: UrlInformation = {
 				ip: urlBody.ip,
 				urlVisitCount: 0,
 				infoVisitCount: 0,
 				lastUsed: new Date().toISOString(),
-			} as UrlWithInformation
+			} as UrlInformation
```

Comment:
You can remove the "as UrlInformation".

In Typescript, `as X` is used to tell Typescript "I don't care what's in that variable, treat it as type X", which is very blunt. Since you added a type to the variable at the start and it's valid, you won't need that `as`.

Reply:
Fixed it, thanks!